mibs/ciena/NortelOpticalMibBundle

```diff
@@ -0,0 +1,16688 @@
+-- Ciena proprietary SNMP MIBs - Common  Bundle - Version 1.0.8.0
+-- (For use with Ciena 6500 and Ciena Common Photonic Layer products)
+-- Packaged on Thu Feb 20 11:19:18 IST 2020
+
+NORTEL-MIB DEFINITIONS ::= BEGIN
```

Comment:
Hello,

Could you check each MIB file and rename them with their official name (the string before DEFINITIONS ::= BEGIN, which is "NORTEL-MIB" in thatcase)

Reply:
And only include the mibs needed

Reply:
I have removed the unused mibs .